vars/debian.yml

```diff
@@ -23,3 +23,7 @@ cvmfs_packages:
     - squid
   client:
     - cvmfs
+  publisher:
+    - apache2
```

Comment:
Why do we need `apache2` on the publisher?

Reply:
yes I just checked in all the docs including the official document it does not have an http package so I will delete it

Reply:
fixed